docs/pages/admin-guides/management/operations/ca-rotation.mdx

```diff
@@ -28,23 +28,32 @@ following order:
 1. `init`: A new certificate authority is issued, but not used.
 1. `update_clients`: The Teleport Auth Service uses the new CA to sign
    certificates but continues to trust certificates signed by the original CA.
-1. `update_servers`: Teleport cluster components (Agents, Auth Service, and
-   Proxy Service instances) reload and start serving TLS and SSH certificates
-   signed by the new certificate authority, but still accept certificates issued
-   by the original certificate authority. This only applies to the Teleport
-   [`host` CA](#host).
-   1. `standby`: No rotation in progress. All operations have completed.
+1. `update_servers`: Any server components in the cluster that accept incoming
+   connections from clients should reload their identity and start serving
+   certificates issed by the new CA, but still accept certificates issued by the
+   original CA.
+   When rotating the [`host` CA], this automatically applies for teleport
+   cluster components (Agents, Auth Service, and Proxy Service instances).
```

Comment:
```suggestion
   cluster components (Agents, Auth Service and Proxy Service instances).
```
Since "instances" modifies "Auth Service" and "Proxy Service"